openquake/qa_tests_data/disagg/case_4/job.ini

```diff
@@ -1,6 +1,6 @@
 [general]
 
-description = Disaggregation Hazard Demo
+description = Disaggregation with sampling
```

Comment:
"with sampling" refers to what? could you be more specific here

Reply:
There is only one kind of sampling, i.e. setting number_of_logic_tree_samples > 0. No room for confusion.